net/adguardhome/files/adguardhome.init

```diff
@@ -21,13 +21,14 @@ start_service() {
   fi
 
   config_load adguardhome
+  config_get CONFIG_FILE config config
```

Comment:
Why did you remove the fallback value `"/etc/adguardhome.yaml"`

Reply:
because default config has value for it?

Reply:
if fallback values are necessary for must-have arguments, even they have values in default config, I should add fallback values for `config_get WORK_DIR config workdir` instead